render-test/runner.cpp

```diff
@@ -825,9 +975,9 @@ bool TestRunner::runOperations(const std::string& key, TestMetadata& metadata) {
 }
 
 TestRunner::Impl::Impl(const TestMetadata& metadata)
-    : frontend(metadata.size, metadata.pixelRatio),
+    : frontend(metadata.size, metadata.pixelRatio, metadata.mapMode == MapMode::Continuous),
       map(frontend,
-          mbgl::MapObserver::nullObserver(),
+          mapPassthroughObserver,
```

Comment:
Before the change, we used nullobserver (stateless), now `mapPassthroughObserver` has state and might need to be 'reset' between test runs.

Reply:
That is true. The example observer `MapBecomeIdleObserver` uses RAII to init and deinit the passthrough observer. The preferred way would have been to add `setObserver()` and `getObserver()` to the map class, but I didn't dare to change the interface.